iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/setup/DirectMethodsCommon.java

```diff
@@ -228,6 +244,98 @@ public void openDeviceClientAndSubscribeToMethods() throws Exception
             null);
     }
 
+    public void SubscribeToMethodAndReceiveAsJsonString() throws Exception
+    {
+        testInstance.setup();
+        this.testInstance.identity.getClient().open(true);
+
+        this.testInstance.identity.getClient().subscribeToMethods(
+                (methodName, methodData, context) ->
+                {
+                    String methodDataAsJsonString = methodData.getPayloadAsJsonString();
+
+                    System.out.println("Device invoked " + methodName);
```

Comment:
We don't typically log something like this using System.out.println. Try using ```log.info(...)``` if you feel this is still worth logging